<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>457</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="Tab">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Theme">
       <attribute name="title">
        <string>Theme</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Themes :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Theme">
            <item>
             <property name="text">
              <string>Light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Classic Dark</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Classic Light</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>custom ...</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_Theme">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>283</width>
             <height>312</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Custom Theme</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_Theme">
                <property name="text">
                 <string>Classic</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Background : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Background">
                <item>
                 <property name="text">
                  <string>White</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gray</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Black</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bright Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bright Purple</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark Purple</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Yellow</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orange</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Base :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Base">
                <item>
                 <property name="text">
                  <string>White</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gray</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Black</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bright Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bright Purple</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dark Purple</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Yellow</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orange</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>WindowText : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_WindowsText">
                  <item>
                   <property name="text">
                    <string>White</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Gray</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Green</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Yellow</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Red</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Orange</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Text : </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_Text">
                  <item>
                   <property name="text">
                    <string>White</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Gray</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Green</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Yellow</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Red</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Orange</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_22">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Button :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_Button">
                  <item>
                   <property name="text">
                    <string>White</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Gray</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Green</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Yellow</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Red</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Orange</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_23">
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>ButtonText :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_ButtonText">
                  <item>
                   <property name="text">
                    <string>White</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Gray</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Green</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Yellow</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Red</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Orange</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Slider Color :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_Slider">
                  <item>
                   <property name="text">
                    <string>White</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Gray</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Black</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Blue</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Bright Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Dark Purple</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Green</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Yellow</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Red</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Orange</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Slider Size</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_Slider"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Icon : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Icon">
                <item>
                 <property name="text">
                  <string>Type 1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Type 2</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Type 3</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Text Font :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_Font">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Arial</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MS Shell Dlg 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Segoe Script</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Zar</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Bold">
              <property name="maximumSize">
               <size>
                <width>45</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Bold</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Text Size :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_size">
              <property name="minimumSize">
               <size>
                <width>172</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PlayBack">
       <attribute name="title">
        <string>PlayBack</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Speed : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
